kinto/__main__.py

```diff
@@ -23,7 +23,8 @@ def main(args=None):
 
     parser = argparse.ArgumentParser(description="Kinto Command-Line "
                                                  "Interface")
-    commands = ('init', 'start', 'migrate', 'delete-collection', 'version')
+    commands = ('init', 'start', 'migrate', 'delete-collection', 'version',
+                'rebuild-quotas')
```

Comment:
I don't think this would be possible, because to know that you would need to read the configuration file which is already a parameter of the CLI tool...